--- xrdp-0.10.0-beta.1/sesman/libsesman/sesman_config.c.o	2024-03-11 14:21:33.295277249 +1100
+++ xrdp-0.10.0-beta.1/sesman/libsesman/sesman_config.c	2024-03-11 14:24:16.114048106 +1100
@@ -249,14 +249,14 @@ config_read_globals(int file, struct con
         cf->default_wm = g_strdup("startwm.sh");
     }
     /* if default_wm doesn't begin with '/', it's a relative path to
-     * XRDP_CFG_PATH */
+     * /usr/libexec/xrdp */
     if (cf->default_wm[0] != '/')
     {
         /* sizeof operator returns string length including null terminator  */
-        int length = (sizeof(XRDP_CFG_PATH) +
+        int length = (sizeof("/usr/libexec/xrdp") +
                       g_strlen(cf->default_wm) + 1); /* '/' */
         char *buf = (char *)g_malloc(length, 0);
-        g_sprintf(buf, "%s/%s", XRDP_CFG_PATH, cf->default_wm);
+        g_sprintf(buf, "%s/%s", "/usr/libexec/xrdp", cf->default_wm);
         g_free(cf->default_wm);
         cf->default_wm = buf;
     }
@@ -267,14 +267,14 @@ config_read_globals(int file, struct con
         cf->reconnect_sh = g_strdup("reconnectwm.sh");
     }
     /* if reconnect_sh doesn't begin with '/', it's a relative path to
-     * XRDP_CFG_PATH */
+     * /usr/libexec/xrdp */
     if (cf->reconnect_sh[0] != '/')
     {
         /* sizeof operator returns string length including null terminator  */
-        int length = (sizeof(XRDP_CFG_PATH) +
+        int length = (sizeof("/usr/libexec/xrdp") +
                       g_strlen(cf->reconnect_sh) + 1); /* '/' */
         char *buf = (char *)g_malloc(length, 0);
-        g_sprintf(buf, "%s/%s", XRDP_CFG_PATH, cf->reconnect_sh);
+        g_sprintf(buf, "%s/%s", "/usr/libexec/xrdp", cf->reconnect_sh);
         g_free(cf->reconnect_sh);
         cf->reconnect_sh = buf;
     }
